Packs/AzureSentinel/Scripts/MicrosoftSentinelSetOwner/MicrosoftSentinelSetOwner.py

```diff
@@ -15,41 +15,38 @@ def set_owner(context_results: dict):
                          and the updated owner information.
     """
     args = demisto.args()
-    mirror_dir = next(
-        (
-            tag["value"]
-            for tag in context_results.get("labels", [])
-            if tag.get("type") == "mirror_direction"
-        ),
-        context_results.get("dbotMirrorDirection"),
-    )
-    incident_id = dict_safe_get(
+    incident_id = args.get("incident_id") or dict_safe_get(
         context_results, ["CustomFields", "sourceid"], ""
-    ) or args.get("incident_id")
+    )
     instance_name = context_results.get("sourceInstance") or args.get("using")
     user_principal_email = args.get("user_principal_name")
-    if mirror_dir in ["null", "Out"]:
-        return CommandResults(
-            readable_output="""The 'Owner' field has not been changed.
-            The 'Owner' field can only be modified when the Mirroring Direction is set to 'Incoming' or 'Incoming and Outgoing'."""  # noqa: E501
+    if not instance_name:
+        return_error(
+            message="Please provide a not empty 'using' as an argument when executing the script from the War Room."
```

Comment:
Same for the email and incident id and the other scripts.
```suggestion
            message="Missing instance name. Make sure to provide a non-empty 'using' argument when executing the script from the War Room."
```